### ISSUE TITLE
"No DecSync dir configured after F-droid privileged extension installation and 1.1.10->1.2.0 upgrade

### ISSUE THREAD

**Issue by jjakob** [39aldo39/DecSyncCC#23]
Today I installed the F-droid privileged extension. After a reboot, I upgraded the app from within F-Droid. Now both my sync accounts were lost and the app thinks it's been clean installed. Was this caused by me installing the privileged extension? How do I get the accounts back with my old data without losing anything?

**Comment by 39aldo39** [comment]
It wasn't caused by the privileged extension, it's just the upgrade to 1.2.0. This version updated some interface with Android, which requires you to reselect the DecSync directory. Your sync accounts should still be there. And even if they somehow got removed, the data is still in the DecSync directory.

However, this new interface with Android doesn't work well yet, so I reverted the change in the upcoming version 1.2.1. Some parts of version 1.2.0 work very, very slow, which even causes some operations to not complete.

So, if you wait for version 1.2.1 to arrive in F-Droid it should be back to normal.
